### PR TITLE
CAMEL-15266: Contributing guide cleanup

### PR DESCRIPTION
**Pull request by AemieJ**

### CODE REVIEW

File: docs/user-manual/modules/ROOT/pages/contributing.adoc [hunk 1]
```diff
@@ -169,7 +170,7 @@ We gladly accept patches if you can find ways to improve, tune or fix Camel in s
 
 We recommend using github PRs instead of manual patch files. Especially for bigger patches.
 
-Most IDEs can create nice patches now very easily. e.g. in Eclipse just right click on a file/directory and select Team \-> Create Patch. Then just save the patch as a file and attach it to the corresponding issue on our https://issues.apache.org/jira/browse/CAMEL[JIRA issue tracker].
+Most IDEs can create nice patches now very easily. e.g. in Eclipse just right click on a file/directory and select Team \-> Create Patch. Then just save the patch as a file and attach it to the corresponding issue on our https://issues.apache.org/jira/browse/CAMEL[JIRA issue tracker]. (You may have to click on Team -> Share... first to enable the Subversion options).
```
Comment (oscerd) [inline docs/user-manual/modules/ROOT/pages/contributing.adoc:173]: I think there is anybody out there still using subversion for Apache Camel.

File: docs/user-manual/modules/ROOT/pages/contributing.adoc [hunk 2]
```diff
@@ -86,6 +86,7 @@ For more information see xref:faq:how-does-the-website-work.adoc[How does the we
 
 Please raise a new issue on our https://issues.apache.org/jira/browse/CAMEL[JIRA issue tracker]. This way we’ll know when it’s really fixed and we can ensure that the problem stays fixed in future releases. Please describe the bug/issue clearly, and add pictures/screenshots if necessary. If you can create a JUnit test case then your issue is more likely to be resolved quicker.
 e.g. take a look at some of the existing https://svn.apache.org/repos/asf/camel/trunk/camel-core/src/test/java/[unit tests cases]
+Then we can add your issue to Subversion and then we'll know when its really fixed and we can ensure that the problem stays fixed in future releases.
```
Comment (oscerd) [inline docs/user-manual/modules/ROOT/pages/contributing.adoc:89]: Ditto

**Comment by AemieJ** [comment]
@oscerd if there is no subversion used, I think it is best to remove it from both the contribution guidelines then?

**Comment by oscerd** [comment]
Yes, I think subversion shouldn't be mentioned at all. I don't know if someone is using it anyway. But I don't think so.

**Comment by oscerd** [comment]
LGTM

**Pull request merged**